src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -268,13 +269,17 @@ const ModalCardDetails = ({
   };
 
   useEffect(() => {
-    setCanSubmit(
+    setCanSubmit(checkFormValidity);
```

Comment:
does `checkFormValidity` need its `()`? functions as variables always confuse me.

Reply:
This might fix your build issue -- not 100% sure

Reply:
ah; that seems likely. will commit it.